@ -1,162 +1,152 @@
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecetaWindow</class>
 <widget class="QMainWindow" name="RecetaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Receta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="volverButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: lightgray;
    color: black;
    border: 2px solid gray;
    border-radius: 10px;
    padding: 5px;
}

QPushButton:hover {
    background-color: silver;
}

QPushButton:pressed {
    background-color: darkgray;
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aceptarButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: lightgreen;
    color: Black;
    border: 2px solid #006400;
    border-radius: 10px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #00b300;
}

QPushButton:pressed {
    background-color: #004d00;
}</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <y>40</y>
      <width>352</width>
      <height>139</height>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="treatmentLabel">
       <property name="text">
        <string>Tratamiento:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
     <item row="0" column="1">
      <widget class="QLineEdit" name="treatmentInput"/>
     </item>
     <item row="2" column="0">
     <item row="1" column="0">
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Fecha de Inicio:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
     <item row="1" column="1">
      <widget class="QDateEdit" name="startDateInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <x>400</x>
      <y>60</y>
      <width>331</width>
      <height>139</height>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="pacientLabel">
       <property name="text">
        <string>Paciente:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="pacientComboBox"/>
      <widget class="QComboBox" name="pacientInput"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="finalizedLabel">
       <property name="text">
        <string>Finalizada:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="finalizedCheckBox"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
